test/parallel/test-fs-watchfile.js

```diff
@@ -1,39 +1,48 @@
+// Flags: --expose_internals
 'use strict';
-
 const common = require('../common');
 const fs = require('fs');
 const path = require('path');
 const assert = require('assert');
+const e = require('internal/errors');
 
+const expectedErr = common.expectsError({
+  type: Error,
```

Comment:
Please only use `common.expectsError()` with errors that have been migrated to `internal/errors`. The reason is so that the existing places in the tests where the error message is checked can be more easily found when doing the migration. If the tests already use `common.expectsError()`, then it's far more likely that whoever is doing the migrating will miss those.